src/test/resources/constraintType/expected/com/example/Simple.java

```diff
@@ -1,5 +1,6 @@
 package com.example;
 
+import java.util.Set;
```

Comment:
`Set` isn't used in this example. While there's not necessarily _harm_ in unnecessary imports, I'd prefer that Specimin not generate them.

Removing unnecessary imports is something that IDEs and formatters already do well, though. Maybe the simplest solution is to run an automated formatter (e.g., [google-java-format](https://github.com/google/google-java-format)) over Specimin's output automatically?